google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BlobSnippets.java

```diff
@@ -173,6 +173,22 @@ public Blob copyToStrings(String bucketName, String blobName) {
   }
 
   /**
+   * Example of moving a blob to a different bucket with a different name.
+   */
+  public Blob moveTo(Blob srcBlob, String destBucket, String destBlob) {
```

Comment:
> Clearly I went with the first option, which seemed the simplest.

This is a fine option to me, I suggest you do that by creating a new `BlobSnippets` class instead of passing a `blob` parameter, something like:

```Java
BlobSnippets moveBlobSnippets = new CopyBlobSnippets(copiedBlob);
copiedBlob = moveBlobSnippets.moveTo(copiedBlob, BUCKET, "moveBlob");
assertNotNull(copiedBlob);
copiedBlob.delete();
```
> I wasn't sure if this sample was super relevant for the javadocs

Yeah it's not super relevant but we can put it on top of `copyTo` and it won't harm anybody :)